kratos/geometries/quadrilateral_2d_8.h

```diff
@@ -1539,16 +1509,20 @@ template<class TPointType> class Quadrilateral2D8
     {
```

Comment:
shouldn't you call this function in the other places where you need to compute the shape function values? it is static...designed exactly for this